openbas-front/src/admin/components/scenarios/Scenarios.tsx

```diff
@@ -194,22 +194,14 @@ const Scenarios = () => {
     exportFileName: `${t('Scenarios')}.csv`,
   };
 
-  // Duplicate
-  const [openDuplicateId, setOpenDuplicateId] = useState<string | null>(null);
-  const handleOpenDuplicate = (scenarioId: string) => {
-    setOpenDuplicateId(scenarioId);
-  };
-  const handleCloseDuplicate = () => {
-    setOpenDuplicateId(null);
-  };
+  const [modificationTrigger, setModificationTrigger] = useState<boolean>(false);
 
-  // Export
-  const [openExportId, setOpenExportId] = useState<string | null>(null);
-  const handleOpenExport = (scenarioId: string) => {
-    setOpenExportId(scenarioId);
-  };
-  const handleCloseExport = () => {
-    setOpenExportId(null);
+  const refreshScenarios = () => {
+    searchScenarios(searchPaginationInput).then((result) => {
+      const { data } = result;
+      setScenarios(data.content);
+      setModificationTrigger((prev) => !prev);
```

Comment:
I don't understand the usage of modificationTrigger

Reply:
modificationTrigger just allows me to signal a change of state which I transmit to the PaginationComponent component